daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/ValueTranslatorSpec.scala

```diff
@@ -280,10 +314,22 @@ class ValueTranslatorSpec
       val failure = Failure(Error.Preprocessing.IllegalContractId.NonSuffixV1ContractId(illegalCid))
 
       forEvery(testCasesForCid(legalCid))((typ, value) =>
-        Try(valueTranslator.unsafeTranslateValue(typ, value)) shouldBe a[Success[_]]
+        Try(
+          valueTranslator.unsafeTranslateValue(
+            typ,
+            value,
+            Config.Legacy,
+          )
+        ) shouldBe a[Success[_]]
       )
       forEvery(testCasesForCid(illegalCid))((typ, value) =>
-        Try(valueTranslator.unsafeTranslateValue(typ, value)) shouldBe failure
+        Try(
+          valueTranslator.unsafeTranslateValue(
+            typ,
+            value,
+            Config.Legacy,
```

Comment:
Appreciate we need to support legacy translation but do we need tests to support the new non-legacy modes ?

Reply:
We need to adapt the tests that @nickchapman-da write for importValue